static/src/javascripts/projects/common/modules/commercial/contributions-utilities.spec.js

```diff
@@ -26,11 +26,6 @@ const rawTest = {
             showTicker: false,
             showReminderFields: null,
             backgroundImageUrl: '',
-            maxViews: {
```

Comment:
this test was wrong anyway - `maxViews` is sent by the Epic Tool at the test-level